### ISSUE TITLE
No permission prompt on Android 10

### ISSUE THREAD

**Issue by freebrowser1** [bk138/droidVNC-NG#75]
If you'd like to put out an incentive for fixing this bug, you can do so at https://issuehunt.io/r/bk138/droidVNC-NG

**Describe the bug**
Version 1.2.6 does not work anymore.

**To Reproduce**
1. Install 1.2.6.
2. No prompt appears to set permissions for screen capture and file access.
3. 1.2.6 cannot be installed over 1.2.3.

**Expected Behavior**
It should install over 1.2.3 and, after removing 1.2.3 and do a clean install of 1.2.6, permissions cannot be set. 

**Your environment (please complete the following information):**
 - Android 10.3 (Oneplus 6)
 - VNC viewer and version: RealVNC.

**Comment by bk138** [comment]
Does a clean install of v1.2.6 work for you?

**Comment by freebrowser1** [comment]
I did a clean install after uninstalling 1.2.3 as I was unable to upgrade to 1.2.6.

**Comment by bk138** [comment]
Cannot reproduce. Can you try with another device?

**Comment by freebrowser1** [comment]
I tried again.
Install 1.2.6 over 1.2.3 issues a message 'not installed' as in my previous attempt.
So I did:
* Uninstall 1.2.3.
* Install 1.2.6 and now permission prompts do occur.
To prevent an extra prompt at every start, I entered 
`adb shell cmd appops set net.christianbeier.droidvnc_ng PROJECT_MEDIA allow` 
once as stated in the Github documentation and the prompt is gone and it does work.
So this ticket can be closed.

It works great, but keyboard entry from the computer does not work yet. I use RealVNC for macOS as a VNC viewer.

**Comment by bk138** [comment]
For keyboard please subscribe to #4.

**Comment by freebrowser1** [comment]
Well, I tried again on an Android emulator (Android Studio, running Android natively on ARM as it is a Macbook M1).
The emulated device runs Android 12 (API 31) and the same symptoms appear on 1.2.6.
I had to restart droidVNC two times before it persistenly accepts File access and input access.

When I enabled input access via 
`adb shell settings put secure enabled_accessibility_services net.christianbeier.droidvnc_ng/.InputService`
then another app (Rotation service) stopped working and its Accessibility permission got revoked so I had to grant permission again. Probably an Android bug ?
